File: executable/python/box-plot-jitter.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('test-runner', '9h29fe3l0x')

data = Data([
    Box(
        y=[0, 1, 1, 2, 3, 5, 8, 13, 21],
        boxpoints='all',
        jitter=0.3,
        pointpos=-1.8
    )
])
plot_url = py.plot(data, filename='box-plot-jitter')

File: executable/julia/box-plot-jitter.jl
using Plotly
Plotly.signin("test-runner", "9h29fe3l0x")

data = [
  [
    "y" => [0, 1, 1, 2, 3, 5, 8, 13, 21], 
    "type" => "box", 
    "boxpoints" => "all", 
    "jitter" => 0.3, 
    "pointpos" => -1.8
  ]
]

response = Plotly.plot([data], ["filename" => "box-plot-jitter", "fileopt" => "overwrite"])
plot_url = response["url"]